common/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 0.9.0
+version: 0.9.1
```

Comment:
Because of the added feature it has to be a `0.10.0`.

Reply:
sure, changed the version